src/core/storage.ts

```diff
@@ -41,7 +41,7 @@ export default class Storage {
             const currentResourceState = newState[payload.name] ? { ...newState[payload.name] } : { byId: {}, allIds: [] }
             currentResourceState.byId[payload.id] = payload.resource
             currentResourceState.allIds = Object.keys(currentResourceState.byId)
-            //TODO: debug or dev mode log?
+            // TODO: debug or dev mode log?
```

Comment:
can use `consola` module and also conditional for not in production could be `process.env.NODE_ENV !== 'production'` I think? Otherwise we can get `isDev` from the nuxt context and pass it to this class.

Reply:
isDev is better or else it'll be available through ssr only

Reply:
Looks pretty straight forward now I've seen again. `ctx` already in this class, so can just pass `ctx.isDev` into the `_initState` function.

Reply:
Sorry ignore that, consola log levels can be configured in each runtime as wanted so probably no need for the dev check.